### DISCUSSION TITLE
JSONDecode Error

### DISCUSSION THREAD

**Discussion by itsjiaming** [kevinzg/facebook-scraper#803] [General]
Hi, I'm facing an issue where I'm either not getting any data from scraping (empty dataset) or I'm getting a JSONDecode Error, depending on which cookies I use. A screenshot of my code with the error is shown below. Has anyone faced the same issue as well? Any help would be appreciated, thanks!
<img width="631" alt="image" src="https://user-images.githubusercontent.com/72295051/179135543-3f2275c4-7a64-481a-a613-84fb2ebf99ad.png">

**Comment by neon-ninja** [comment]
Could you please expand those 6 frames?
